examples/pxScene2d/src/pxShaderResource.cpp

```diff
@@ -637,8 +745,6 @@ void pxShaderResource::loadResource(rtObjectRef archive, bool reloading)
     mDownloadInProgressMutex.unlock();
     AddRef(); //ensure this object is not deleted while downloading
     rtFileDownloader::instance()->addToDownloadQueue(vtxRequest);
-
-    return;
   }
   else if (isFrgDataURL)
```

Comment:
Should this be an `if` instead of a `else if` if there is both a vertex shader and a fragment shader data url?  If so, check the following `else if` and `else` statements as the logic may need to be updated.  If the intention is to not support a vertex shader and a fragment shader data url with the same resource then this code doesn't need to be modified.  However, I just wanted to point this out in case that was not the intention.

Reply:
Reworked. And added additional testing to check permutations.